### PR TITLE
Add Footer and Navbar placeholders to layout

### PR DESCRIPTION
**Pull request by BruceJi7**

### CODE REVIEW

File: src/components/Layout/Footer/Footer.module.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.Footer {
+  background-color: bisque;
```
Comment (ivanms1) [inline src/components/Layout/Footer/Footer.module.scss:2]: can you make this a variable?

File: src/components/Layout/index.tsx [hunk 2]
```diff
@@ -1,13 +1,22 @@
 import React from "react";
 
+import Navbar from "./Navbar";
+
 import styles from "./Layout.module.scss";
+import Footer from "./Footer";
```
Comment (ivanms1) [inline src/components/Layout/index.tsx:6]: import order

**Review by ivanms1 — CHANGES_REQUESTED** [review]

File: src/components/Layout/Footer/Footer.module.scss (hunk 1 above)
Comment (BruceJi7) [reply src/components/Layout/Footer/Footer.module.scss:2]: It's only that colour as a temporary measure

File: src/components/Layout/index.tsx (hunk 2 above)
Comment (BruceJi7) [reply src/components/Layout/index.tsx:6]: Crap lol

**Review by ivanms1 — APPROVED** [review]

**Pull request merged**